frontend/types/src/components/Examination/ExaminationForm.vue.d.ts

```
@@ -0,0 +1,141 @@
+    selectedMorphologyClassificationId: import("vue").Ref<number | null, number | null>;
+    filteredMorphChoices: import("vue").ComputedRef<import("@/stores/examinationStore").MorphologyClassificationChoice[]>;
+    loading: import("vue").Ref<boolean, boolean>;
+    error: import("vue").Ref<string | null, string | null>;
+    morphologyClassifications: import("vue").Ref<{
+        id: number;
+        name: string;
+    }[], import("@/stores/examinationStore").MorphologyClassification[] | {
+        id: number;
+        name: string;
+    }[]>;
+}, {}, {}, {}, import("vue").ComponentOptionsMixin, import("vue").ComponentOptionsMixin, {}, string, import("vue").PublicProps, Readonly<{}> & Readonly<{}>, {}, {}, {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)